src/test/unit_tests.cpp

```diff
@@ -18,6 +18,7 @@
 
 int main(int argc, char** argv) {
     ::testing::InitGoogleTest(&argc, argv);
+    ::testing::InitGoogleMock(&argc, argv);
```

Comment:
Why wasn't it required before?

Reply:
It just enables you to use those arguments. In my case it was using `--gmock_verbose=info`.

Without the Init it was just not passed into GMock. If you won't do that then nothing happens.